changelog/rdmdMakefiles.dd

```diff
@@ -0,0 +1,12 @@
+rdmd can now be used as a shell in makefiles
+
+With gnu make(3.82 or higher), rdmd can now be used in makefiles.
+This is accomplished by setting the SHELL and .SHELLFLAGS to /usr/bin/rdmd and --eval respectively.
+---
+.ONESHELL:
+SHELL = /usr/bin/rdmd
+.SHELLFLAGS = --eval
+hello.txt:
+	import std.file;
```

Comment:
We should really resolve this conflict between std.stdio.write and std.file.write.
There's one workaround though: `"$@".File("w").writeln("hello world");`, but I'm not sure if it's better than the import.